### ISSUE TITLE
Create rancher env from dashboard not working

### ISSUE THREAD

**Issue by coolvijay89** [SocketCluster/baasil-cli#2]
Hello John,

I would like to use baasil, kubernets and dockers. But I couldn't able to create rancher environment from Baasil dashboard.

So I have installed rancher, Kubernets, and dockers and run the create baasil app and it run successfully on 8000 port.

But when I have tried to deploy the app it show me the message that app has been deployed successfully but when I run the URL as shown in the Baasil Video then its not loading the app over there.

So can you please guide me what I am doing wrong?

Thanks for your help in advance.

Thanks,
Vijay.

**Comment by jondubois** [comment]
@coolvijay89 Did you edit your `~/.kube/config` file to point to the correct Kubernetes environment?
Can to copy this file from Rancher or Kubernetes dashboard.

**Comment by coolvijay89** [comment]
@jondubois Yes I have edited the ~/.kube/config file to point to the correct Kubernetes environment.
I have copied that file from Rancher dashboard.

**Comment by coolvijay89** [comment]
@jondubois We would like to use Baasil IO. Can you please help us to set up that? We have already configured the credit card in the Baasil.

**Comment by jondubois** [comment]
@coolvijay89 Please provide your email through this form https://baasil.io/#!/contact-us so that we can get in touch.

**Comment by coolvijay89** [comment]
@jondubois I have sent an email. I have sent an email before some days ago but haven't received any feedback from your team yet. Also can you please give me your thoughts on why that baasil app not deploying on the IP address?
